leap/example.rb

```diff
@@ -1,25 +1,12 @@
 class Year
```

Comment:
The existing solution was very wordy, with attr_reader, initialise, etc. Happy to revert to the original solution if it's better style.